chart/env/dev.yaml

```diff
@@ -166,7 +166,7 @@ configNames:
       cpu: 1
       memory: "4Gi"
 
-splitNames:
+splitNamesStreaming:
```

Comment:
I know that we should send another PR for chart folder but since this change is dependent on the worker name I just did the change of the names (Only names, nothing related to infrastructure configuration)

Reply:
Hmmm, I think it's better to keep with the logic that: if we don't update the docker images, we don't touch the chart. When we do a new release, the list of changes will make it clear what refactoring should be done in chart/. wdyt?

Reply:
Got it, I removed this changes